bindings/protocols/bacnet/index.html

```diff
@@ -770,6 +792,10 @@ <h3>WoT Data Schema and BACnet Types</h3>
                                     "$comment": "ISO8601 Time Format with optional wildcards"
                                 }
                             </code>
+
+                            The wildcards are used to indicate occurrences,
+                            e.g. <code>*:00:00Z</code> to indicate all seconds of the first minute of the hour in UTC.
```

Comment:
This doesn't make sense to me. I see "the zero'th second of every hour". My suggestion *should* indicate every second of the first minute of each hour.
(Also, there's a missing comma.)
```suggestion
                            Wildcards are used to indicate occurrences,
                            e.g., <code>*:00:*Z</code> to indicate all seconds of the first minute of each hour in UTC.
```

Reply:
Thank you, this was a mistake. Now corrected with the suggestion from @ektrah.